src/components/Persona/Persona.md

```diff
@@ -0,0 +1,71 @@
+# Persona
+Represents a person, complete with a profile image and additional details.
```

Comment:
We should be moving to the Initials, but it may be worth noting the fallback behavior when a user's image is unavailable.